test/integration/puppetlabs/services/certificate_authority/ca_true_test.clj

```diff
@@ -0,0 +1,34 @@
+(ns puppetlabs.services.certificate-authority.ca-true-test
+  (:require [clojure.test :refer :all]
+            [clojure.tools.logging :as log]
+            [puppetlabs.puppetserver.bootstrap-testutils :as bootstrap]
+            [puppetlabs.http.client.sync :as http-client]
+            [clojure.pprint :as pprint]
+  )
+)
+
+(deftest ca-true-test
+  (testing "We are validating that a csr with the CA:TRUE extension is rejected for signing by the server"
+      (bootstrap/with-puppetserver-running
+        app
+        {:jruby-puppet
+        {:master-conf-dir "./dev-resources/puppetlabs/ca_true_test/master/conf"}}
+        (let
+          [response (http-client/put (str "https://localhost:8140/"
```

Comment:
It might be good to move this `(str ...` onto a new line - it will change the indenting so that it's 1 or 2 spaces indented from `(http-client...` which will indent the below map less too (because it tries to align all arguments) - that lines 19-21 won't extend too far off the page.